x/tss/keeper/querier.go

```diff
@@ -531,3 +534,23 @@ func QueryExternalKeyID(ctx sdk.Context, k types.TSSKeeper, n types.Nexus, chain
 
 	return types.ModuleCdc.MarshalLengthPrefixed(&resp)
 }
+
+// QueryGovernanceKey returns the governance key
+func queryGovernanceKey(ctx sdk.Context, k types.TSSKeeper) ([]byte, error) {
+	governanceKey, ok := k.GetGovernanceKey(ctx)
+	if !ok {
+		return nil, fmt.Errorf("governance key not set")
+	}
+
+	var pubKeys []string
+	for _, pk := range governanceKey.GetPubKeys() {
+		bz, _ := types.ModuleCdc.MarshalInterfaceJSON(pk)
+		pubKeys = append(pubKeys, string(bz))
```

Comment:
What does `string(bz)` look like?

Reply:
it converts to
```
'{"type":"tendermint/PubKeySecp256k1","value":"AtSAqsCaMKkDFTc5nVMz0CpPdDdKXlcP0IFh7+Y+wo5p"}'
```
I changed it return the multisig LegacyAminoPubKey after switched to grpc query